packages/server/graphql/mutations/helpers/generateWholeMeetingSummary.ts

```diff
@@ -33,7 +33,12 @@ const generateWholeMeetingSummary = async (
   if (contentToSummarize.length <= 1) return
   const [r, summary] = await Promise.all([getRethink(), manager.getSummary(contentToSummarize)])
   if (!summary) return
-  await r.table('NewMeeting').get(meetingId).update({summary}).run()
+  const [meeting] = await Promise.all([
+    dataLoader.get('newMeetings').load(meetingId),
+    r.table('NewMeeting').get(meetingId).update({summary}).run()
+  ])
+  // mutate the cache
+  meeting.summary = summary
```

Comment:
Need to update the cache for the notifications to have access to the summary

Reply:
Not ideal, but probably the best. Let's add the load call to the `Promise.all` in l13.

As updating the cache is more and more common, it would be nice to extend the Dataloader with an `update` function where you can update the object (if it is loaded already).